packages/lisk-core/src/channels/child_process.js

```diff
@@ -0,0 +1,108 @@
+const { EventEmitter2 } = require('eventemitter2');
+const axon = require('axon');
+const rpc = require('axon-rpc');
+const Promise = require('bluebird');
+const Action = require('../action');
+const Event = require('../event');
+const config = require('../helpers/config');
+const BaseChannel = require('./base');
+
+module.exports = class ChildProcessChannel extends BaseChannel {
```

Comment:
This is a personal preference but In general I'd like to name in a way that it reflects what it's exporting. We have several `child_process.js` files inside different folders, doing different tasks. I'd prefer to rename it `ChildProcessChannel.js` since it immediately makes easy to understand what this file is exporting.